warehouse/packaging/tasks.py

```diff
@@ -56,6 +56,21 @@ def sync_file_to_cache(request, file_id):
         file.cached = True
 
 
+@tasks.task(ignore_result=True, acks_late=True)
+def compute_packaging_metrics(request):
+    metrics = request.find_service(IMetricsService, context=None)
+
+    metrics.gauge(
+        "warehouse.packaging.total_projects", request.db.query(Project).count()
```

Comment:
The .count() operator isn't as efficient as using the SQL func.count() - since this query selects all fields and then runs a count on them.
See this example for a more efficient approach:
https://github.com/pypi/warehouse/blob/6be6bccf07dace18784ea8aeac7906903fdbcf3a/warehouse/accounts/tasks.py#L33

Reply:
Even better is to use the `RowCount` table :) 

https://github.com/pypi/warehouse/blob/main/warehouse/views.py#L255